### PR TITLE
docs(guide): convert issue-help to rst

### PR DESCRIPTION
**Pull request by daprahamian**
Fixes NODE-2181

## Description

**What changed?**

**Are there any files to ignore?**

### CODE REVIEW

File: docs/guide/issues-help.txt [hunk 1]
```diff
@@ -1,3 +1,59 @@
...
+To get started check out the source and work on a branch:
+
+.. code-block:: bash
+
+   git clone https://github.com/mongodb/node-mongodb-native.git
+   cd node-mongodb-native
+   npm install
+   git checkout -b myNewFeature
+
+To run the test suite, you must have the ``mongod`` executable in your current
```
Comment (daprahamian) [inline docs/guide/issues-help.txt]: These eventually need to be updated to the new instructions.

Comment (mbroadst) [reply docs/guide/issues-help.txt]: is it worth just updating this now? By the time this is released it will be the new way

File: docs/guide/issues-help.txt [hunk 2]
```diff
@@ -1,3 +1,59 @@
 =============
 Issues & Help
-=============
\ No newline at end of file
+=============
+
+Our developer community is vibrant and highly engaged, with wide experience using Node.js with MongoDB.
```
Comment (mongomoe) [inline docs/guide/issues-help.txt]: Not a mandatory change, but this would read better if it said:

"Our developer community is vibrant and highly engaged, with **extensive** experience using Node.js with MongoDB"

rather than

"Our developer community is vibrant and highly engaged, with **wide** experience using Node.js with MongoDB"

File: docs/guide/issues-help.txt [hunk 3]
```diff
@@ -1,3 +1,59 @@
...
+To report a bug or to request a new feature in the Node.js driver,
+please open a case in our issue management tool, JIRA:
+
+
+* `Create an account and login <https://jira.mongodb.org>`_.
+* Navigate to `the NODE project <https://jira.mongodb.org/browse/NODE>`_.
+* Click **Create Issue**. Provide as much information as possible about the
+  issue and the steps to reproduce it.
+
+Bug reports in JIRA for the Node.js driver and the Core Server (i.e. SERVER) project are **public**.
```
Comment (mongomoe) [inline docs/guide/issues-help.txt]: Here you mentioned "Core Server", do you mean the mongodb server (I think - this repo: https://github.com/mongodb/mongo)? 
If so, can we rephrase this to say:

"Bug reports in JIRA for the Node.js driver and the Server (i.e Core Server) project are public."

The reason for this requested change is because, to my understanding, the mongodb database is typically referred to as "the MongoDB server" rather than "the Core MongoDB server". I don't think it's a big deal which we choose to use for this doc, but the former is probably more clear

File: docs/guide/issues-help.txt (hunk 2 above)
Comment (mongomoe) [reply docs/guide/issues-help.txt]: Please ignore the above comment as it involves a readability change rather than a code/technical change

**Comment by daprahamian** [comment]
@mbroadst I have changed the instructions to reflect the new test-runner.

File: docs/guide/issues-help.txt [hunk 4]
```diff
@@ -1,3 +1,53 @@
...
+To get started check out the source and work on a branch:
+
+.. code-block:: bash
+
+   git clone https://github.com/mongodb/node-mongodb-native.git
+   cd node-mongodb-native
+   npm install
+   git checkout -b myNewFeature
+
+To run the test suite, you must have a server topology running, and provide the URI to the command.
```
Comment (mbroadst) [inline docs/guide/issues-help.txt]: Does it make sense to note that they could e.g. run `tools/cluster_setup.sh`, or also link to things like `mtools` or `mongo-orchestration`?

Comment (daprahamian) [reply docs/guide/issues-help.txt]: We could definitely link to `mtools` or `mongo-orchestration` in an info block. Where would be the best place to link to those? Is there a good place to link to for `pip` packages?

Comment (daprahamian) [reply docs/guide/issues-help.txt]: I guess we could use https://pypi.org/project/mtools/ and https://pypi.org/project/mongo-orchestration/ ?

Comment (mbroadst) [reply docs/guide/issues-help.txt]: SGTM
